src/elements/OcModal.vue

```diff
@@ -213,23 +226,32 @@ export default {
     },
 
     $_ocModal_confirm() {
-      const value = this.hasInput ? this.$refs.ocModalInput.value : null
+      if (this.buttonConfirmDisabled || this.inputError) {
+        return
+      }
 
       /**
        * The user clicked on the confirm button. If input exists, emits it's value
        * 
-       * @property {String} inputValue Value of the input
+       * @property {String} value Value of the input
        */
-      this.$emit("confirm", value)
+      this.$emit("confirm", this.$_ocModal_input_value)
     },
 
-    $_ocModal_input_type() {
+    $_ocModal_input_type(value) {
+      // Assign value to the input value
+      this.$_ocModal_input_value = value
+
       /**
        * The user typed into the input
        * 
-       * @property {String} inputValue Value of the input
+       * @property {String} value Value of the input
        */
-      this.$emit("type", this.$refs.ocModalInput.value)
+      this.$emit("type", value)
+    },
+
+    $_ocModal_input_assignPropAsValue(value) {
+      this.$_ocModal_input_value = value
```

Comment:
Like above, this is already handled through `v-model` on the input. So you could get rid of this handler.

Reply:
This is not getting the value from the input. This assigns the prop called `inputValue` to the input.